Content.Server/GameObjects/Components/Weapon/Melee/UnarmedCombatComponent.cs

```diff
@@ -0,0 +1,12 @@
+﻿
+using Content.Server.GameObjects.EntitySystems;
+using Robust.Shared.GameObjects;
+
+namespace Content.Server.GameObjects.Components.Weapon.Melee
+{
+    [RegisterComponent]
+    public class UnarmedCombatComponent : MeleeWeaponComponent
```

Comment:
You created this class here, but it isn't really used elsewhere in your changes, right?

Reply:
Used in the Human.yml
I could just put the MeleeWeaponComponent on the human, but there should probably be some differences.

Reply:
Ah, I completely missed that this inherited MeleeWeaponComponent--

Reply:
zummkopf